primitives/io/src/lib.rs

```diff
@@ -86,7 +86,13 @@ pub enum EcdsaVerifyError {
 pub trait Storage {
 	/// Returns the data for `key` in the storage or `None` if the key can not be found.
 	fn get(&self, key: &[u8]) -> Option<Vec<u8>> {
-		self.storage(key).map(|s| s.to_vec())
+		let res = self.storage(key).map(|s| s.to_vec());
+		sp_tracing::debug!(
```

Comment:
We had this already in a previous pr of you, I don't see why you add this?

This is already traced by `Ext`.

So please revert all the changes in this file and the `Cargo.toml`.